File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        map<int, int> m;
        for(int i = 0; i < nums.size(); i++) {
            m.insert({nums[i], nums[i]});
        }

        return m.size() != nums.size();
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        cnt = {}
        res = []
        for n in nums:
            cnt[n] = cnt.get(n, 0) + 1
        
        for key,v in sorted(cnt.items(), key = lambda item : item[1], reverse=True):
            if k>0:
                res.append(key)
                k -= 1
            else:
                break
        return res
